docs/docs/mdx/writing-pages.md

```diff
@@ -49,14 +46,34 @@ query {
 }
 ```
 
-## Importing components
+> **Note:** To query `Mdx` content, it must be included in the node system using a
+> source like the `gatsby-source-filesystem` plugin first. Instructions for sourcing
+> content from somewhere like your `/src/pages` directory can be found on the [plugin's README](/packages/gatsby-source-filesystem/).
+
+Frontmatter is also available in `props.pageContext.frontmatter` and
+can be accessed in blocks of JSX in your MDX document:
+
+```mdx
+---
+title: "Building with Gatsby"
+author: "Jay Gatsby"
+---
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<span>{props.pageContext.frontmatter.author}</span>
+
+(Blog post content, components, etc.)
+```
 
-Similarly to what you'd do in JSX, you can import and render components
-with JSX. You can also import other MDX documents.
+## Importing JSX components and MDX documents
```

Comment:
Should this be "Importing JSX components _in_ MDX documents"?

Reply:
I think the "and" is good because it shows that both can be imported rather than just React components. JSX components _and_ MDX documents are able to be imported in an MDX file.

Reply:
ok, we can leave as-is!